File: JAVA/2021/1stOct_yasas4d.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
	
	System.out.println("Enter input number");
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();

        if(n <= 0)
            return;

        if(n == 1) {
            System.out.println("1");
            return;
        }

        int a = 1,b = 1;
        System.out.print(a + " " + b + " ");
        for(int i=0; i<n-2; i++) {
            int c = a + b;
            boolean flag = true;

            for (int j = 2; j * j <= c; j++) {
                if (c % j == 0) {
                    flag = false;
                    break;
                }
            }
            a = b;
            b = c;

            if (flag || c % 5 == 0)
                System.out.print("0 ");
            else
                System.out.print(c + " ");
        }

    }

}

File: Python/2021/1stOct_yasas4d.py
def fibonacci(n):
    if n == 0:
        return 0
    elif n == 1 or n == 2:
        return 1
    else:
        return fibonacci(n-1) + fibonacci(n-2)


def prime(n):
    c = 0
    for i in range(1, n+1):
        if n % i == 0:
            c += 1
    if c == 2:
        return True
    else:
        return False


print("Enter input number")
n = int(input())
for i in range(1, n+1):
    if prime(fibonacci(i)) == True or i % 5 == 0:
        print(0, end=" ")
    else:
        print(fibonacci(i), end=" ")

print("\n")
